examples/js/loaders/DRACOLoader.js

```diff
@@ -0,0 +1,256 @@
+        }
+
+        return function(rawBuffer) {
+          const scope = this;
+          /*
+           * Here is how to use Draco Javascript decoder and get the geometry.
+           */
+          const buffer = new dracoDecoder.DecoderBuffer();
+          buffer.Init(new Int8Array(rawBuffer), rawBuffer.byteLength);
+          const wrapper = new dracoDecoder.WebIDLWrapper();
+
+          /*
+           * Determine what type is this file: mesh or point cloud.
+           */
+          const geometryType = wrapper.GetEncodedGeometryType(buffer);
+          if (geometryType == dracoDecoder.TRIANGULAR_MESH) {
+            //fileDisplayArea.innerText = "Loaded a mesh.\n";
+          } else if (geometryType == dracoDecoder.POINT_CLOUD) {
+            //fileDisplayArea.innerText = "Loaded a point cloud.\n";
+          } else {
+            const errorMsg = "Error: Unknown geometry type.";
+            //fileDisplayArea.innerText = errorMsg;
+            throw new Error(errorMsg);
+          }
+          return scope.convertDracoGeometryTo3JS(wrapper, geometryType, buffer,
+                                                 dracoDecoder);
+        }
+    } )(),
+
+    convertDracoGeometryTo3JS: function(wrapper, geometryType, buffer,
+                                        dracoDecoder) {
+        let dracoGeometry;
+        const start_time = performance.now();
+        if (geometryType == dracoDecoder.TRIANGULAR_MESH) {
+          dracoGeometry = wrapper.DecodeMeshFromBuffer(buffer);
+        } else {
+          dracoGeometry = wrapper.DecodePointCloudFromBuffer(buffer);
+        }
+        const decode_end = performance.now();
+        dracoDecoder.destroy(buffer);
+        /*
+         * Example on how to retrieve mesh and attributes.
+         */
+        let numFaces, numPoints;
+        let numVertexCoordinates, numTextureCoordinates, numAttributes;
+        // For output basic geometry information.
+        let geometryInfoStr;
+        if (geometryType == dracoDecoder.TRIANGULAR_MESH) {
+          numFaces = dracoGeometry.num_faces();
+          geometryInfoStr += "Number of faces loaded: " + numFaces.toString()
+              + ".\n";
+        } else {
+          numFaces = 0;
+        }
+        numPoints = dracoGeometry.num_points();
+        numVertexCoordinates = numPoints * 3;
+        numTextureCoordinates = numPoints * 2;
+        numAttributes = dracoGeometry.num_attributes();
+        geometryInfoStr = "Number of points loaded: " + numPoints.toString()
+            + ".\n";
+        geometryInfoStr += "Number of attributes loaded: " +
+            numAttributes.toString() + ".\n";
+
+        // Get position attribute. Must exists.
+        const posAttId = wrapper.GetAttributeId(dracoGeometry,
+                                                dracoDecoder.POSITION);
+        if (posAttId == -1) {
+          const errorMsg = "No position attribute found in the mesh.";
+          //fileDisplayArea.innerText = errorMsg;
```

Comment:
Do you mind pulling again from draco. They've removed the html element dependencies: https://github.com/google/draco/issues/68